chai/chai.d.ts

```diff
@@ -25,6 +28,28 @@ declare module Chai {
         (target: any, message?: string): Assertion;
     }
 
+    interface ShouldAssertion {
+        equal(val1: any, val2: any, msg?: string): void;
+        Throw: ShouldThrow;
+        throw: ShouldThrow;
+        exist(val: any, msg?: string): void;
```

Comment:
Yeah. I see arg names like this all the time in libraries, but I don't agree with it. Especially when we're dealing with TypeScript, we're trying to get more information out of libraries, not less. These short-hand names can be confusing at times; though, not particularly in this case. I think the longer name serves well in Intellisense.